### PR TITLE
Add atlantic-2 chain info

### PR DESCRIPTION
**Pull request by mj850**
Update the viem chains with atlantic-2 chain information.

Pending confirmation on URLs, ChainIDs and Precompile Addresses etc.

### CODE REVIEW

File: packages/evm/src/providers/viem.ts [hunk 1]
```diff
@@ -67,3 +67,61 @@ export const ARCTIC_1_VIEM_CHAIN: Chain = {
...
+export const ATLANTIC_2_VIEM_CHAIN: Chain = {
+	blockExplorers: {
+		default: {
+			name: 'SeiTrace',
+			url: 'https://seitrace.com'
+		}
+	},
+	// Precompiles are at the same address.
+	contracts: {
+		addressPrecompile: { address: ARCTIC_1_ADDRESS_PRECOMPILE_ADDRESS },
```
Comment (besated) [inline packages/evm/src/providers/viem.ts:103]: These should be updated to use Atlantic-2 precompile addresses unless all chains use the same address. Then we can just refactor all chains to be just ‘ADDRESS_PRECOMPILE_ADDRESS’

**Review by besated — CHANGES_REQUESTED** [review]

File: packages/evm/src/providers/viem.ts (hunk 1 above)
Comment (mj850) [reply packages/evm/src/providers/viem.ts:103]: Ya actually I think they likely should be the same?
https://github.com/sei-protocol/sei-chain/blob/seiv2/precompiles/staking/Staking.sol
Given that the addresses are written into the code itself

Comment (mj850) [reply packages/evm/src/providers/viem.ts]: Just updated so they all just say <>_address_precompile

Comment (mj850) [reply packages/evm/src/providers/viem.ts]: Psu confirmed that addresses are same throughout

File: .changeset/thick-drinks-wave.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@sei-js/evm": minor
```
Comment (besated) [inline .changeset/thick-drinks-wave.md:2]: i think this can be a `patch` instead of `minor`

**Review by besated — APPROVED** [review]

**Pull request merged**